fastapi/openapi/utils.py

```diff
@@ -178,6 +180,19 @@ def get_openapi_path(
                 )
                 if request_body_oai:
                     operation["requestBody"] = request_body_oai
+            # Add logic for callbacks that are under this operation
+            if route.callbacks:
+                callbacks = {}
+                for callback in route.callbacks:
+                    cb_path, cb_security_schemes, cb_definitions, = get_openapi_path(
+                        route=callback, model_name_map=model_name_map
+                    )
+                    logging.error(cb_definitions)
+                    logging.error(cb_path)
+                    callbacks[callback.name] = {callback.path: cb_path}
+                    # TODO: - do we need to add these definitions here?
```

Comment:
@tiangolo I haven’t had a chance to think about this point yet, wanted to draw your attention to it though.

Reply:
Thanks! Yep, it's not needed.